docs/tutorial/a_minimal_working_example.rst

```diff
@@ -0,0 +1,128 @@
+A minimal working example
+=========================
+
+We start with a minimal working example of ALF. The example, as a pure ALF
+configuration file, is located at ``<ALF_ROOT>/alf/examples/tutorial/minimal_example_conf.py``,
+and consists of only 8 lines.
+
+Train and play
+--------------
+
+Let's ignore its content for a moment (see the next section
+:doc:`./understanding_ALF_via_the_minimal_working_example` for an explanation of
+the configuration content), and just focus on how to launch the training,
+interpret the output training messages, and play a trained model.
+
+Train from scratch
+^^^^^^^^^^^^^^^^^^
+
+We can train from scratch by
+
+.. code-block:: bash
+
+    cd <ALF_ROOT>/alf/examples/tutorial
+    python -m alf.bin.train --root_dir /tmp/alf_tutorial1 --conf minimal_example_conf.py
+
+assuming ``/tmp/alf_tutorial1`` doesn't exist or is empty. The training will finish
+in several seconds, but with some informative messages shown in the terminal. First
+of all, you should see a message from ``checkpoint_utils.py`` like
+
+::
+
+    There is no checkpoint in directory /tmp/alf_tutorial1/train/algorithm. Train
+    from scratch
+
+which basically confirms that the training is from scratch and all algorithm parameters
+and states are randomly initialized. Also ``policy_trainer.py`` will output
+message lines like
+
+::
+
+    minimal_example_conf.py -> alf_tutorial1: 0 time=0.046 throughput=5169.30
+
+which has the formatting template as
+
+::
+
+    [conf_file_name] -> [training_root_dir]: [iteration_numer] time=[current_time_per_training_iter] throughput=[current_training_throughput]
```

Comment:
Here we probably want to briefly explain what is a "training iteration"?

Reply:
I'll leave some core concepts to the next section.